vision/module/location.py

```diff
@@ -285,7 +313,7 @@ def _increaseBrightness(self, increase):
 
     ## Input Functions
 
-    def setImg(self, color, depth):
+    def setImg(self, color, depth) -> None:
```

Comment:
> color, depth

Make sure you type function parameters as well. Should be:
color: np.ndarray, depth: np.ndarray